os-server/src/main/resources/messages.properties

```diff
@@ -8,6 +8,7 @@ validation.email.pattern.message=Invalid email format
 validation.email.length.message=Email is too long. Maximum 45 characters are allowed.
 validation.firstName.length.message=First name is too long. Maximum 45 characters are allowed.
 validation.lastName.length.message=Last name is too long. Maximum 45 characters are allowed.
+validation.title.duplicate=The category with the same title already exists
```

Comment:
since you have a specific message for the category, please rename the key as well 
validation.category.title.duplicate.message